packages/ibm-products/src/components/Coachmark/CoachmarkOverlay.js

```diff
@@ -98,15 +107,17 @@ export let CoachmarkOverlay = forwardRef(
         )}
         ref={overlayRef}
         style={styledTune}
+        aria-labelledby={contentId}
+        tabIndex={-1}
         {...getDevtoolsProps(componentName)}
       >
         {isDraggable ? (
           <CoachmarkDragbar onDrag={handleDrag} onClose={onClose} />
         ) : (
           <CoachmarkHeader onClose={onClose} />
         )}
-        <div className={`${blockClass}__body`} ref={ref}>
-          {children}
+        <div className={`${blockClass}__body`} ref={ref} id={contentId}>
+          {getModifiedChildren()}
```

Comment:
`getModifiedChildren()` and `modifyChildren()` are only referenced once each. Can they simply be inlined here?